include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -175,6 +175,22 @@ class LAPACKFullMatrix : public TransposeTable<number>
   void
   rank1_update(const number a, const Vector<number> &v);
 
+  /**
+   * Apply <a href="https://en.wikipedia.org/wiki/Givens_rotation">Givens</a>
+   * rotation @p csr (a triplet of cosine, sine and radius, see
+   * Utilities::LinearAlgebra::givens_rotation())
+   * to this matrix in the plane spanned by the @p i'th and @p k'th unit vectors.
+   * If @p left is <code>true</code>, the rotation is applied from left
+   * and only rows @p i and @p k are affected.
+   * Otherwise, transpose of the rotation matrix is applied from right
+   * and only columns @p i and @p k are affected.
```

Comment:
do you mean to say that it is *also* applied from the right, i.e., from the left *and* right? Or *only* from the right? Might be worth clarifying.